### DISCUSSION TITLE
access_token和refresh_token可以用别的名称吗

### DISCUSSION THREAD

**Discussion by rhine14586** [spencerwooo/onedrive-vercel-index#708] [Q&A] [answered]
一个redis里面只能放一个access_token和refresh_token，现在我如果要部署新页面就要重开一个redis，可以用不同变量名放在同一个redis里吗？比如access_token2和refresh_token2

**Comment by myl7** [comment, marked as answer]
目前不行，但合并 #572 之后就可以了。
最近我也会处理这个 PR（需要对这个 PR 进行一些改动），如果能等一段时间的话可以稍等一下，急需的话可以先在自己的 fork 里合并这个 PR 的改动。

**Reply by myl7** [reply]
好了现在已经可以了
